### DISCUSSION TITLE
HTML+CSS öğreniyorum 1 (update)

### DISCUSSION THREAD

**Discussion by burakta** [ademilter/homepage#38] [Q&A] [answered]
Adem Bey Merhaba,

İlk videoyu izledim, işinizi özenle ve düzenle yaptığınız çok net anlaşılıyor. Evet header'la geçmiş 2 saat ancak temel, çok önemli noktaları vermişsiniz. 
Sizden ricam aradan geçen zamanı da göz önüne alarak anlatımınızda geçen aşağıya aldığım konuların varsa yine sizin pratiğinizdeki güncel karşılıklarını vermeniz. Teşekkürler.

gulpjs                  :
autoprefixer        :
css-scss               :
normalize            : 
formalize             :

**Comment by ademilter** [comment, marked as answer]
merhaba burak,
gulpjs haricinde yazdığın araçlar ve yöntemler hali hazırda hala kullanılan şeyler. 
gulpjs kullanılmıyor gibi anlaşılmasın. gulpjs ile benzer şeyi yapan alternatif araçlar (https://parceljs.org/) çıktı o yüzden yapacağın işe göre seçim yapman gerekiyor.

#### template (html+css) yazıp başka bir yazılımcıya teslim edecek isen gulpjs

- css,html ve js çıktısı sıkıştırılmamış olması gerekiyor ki view render edecek kişi düzgün okuyabilsin

#### eski yöntem jquery ile site yapıp aynı zamanda sass veya typescript kullanacaksan ve bunları yaparken gulpjs gibi config dosyası ile uğraşmayayım diyorsan parceljs

- çıkan dosyalar sıkıştırılmış ve render olmuş olacağı için js dosyası okunamaz durumda olur. burada dikkat edilecek husus iş çıktıktan sonra yayına alınacak gibi düşünmek lazım.

bunların dışında yeni nesil SPA, SSR uygulamalar için zaten react, vue gibi araçların hazır proje dosyaları ile ilerlemek sağlıklı olacaktır.

**Comment by burakta** [comment]
Çok teşekkürler...Saygılar
